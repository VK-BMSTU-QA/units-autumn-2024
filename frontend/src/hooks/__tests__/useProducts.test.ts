import { useProducts } from '../useProducts';
import { renderHook } from '@testing-library/react';
describe('test useProducts hook', () => {
    it('should return products list', () => {
        const { result } = renderHook(() => useProducts());
        expect(result.current).toMatchSnapshot();
    });
});