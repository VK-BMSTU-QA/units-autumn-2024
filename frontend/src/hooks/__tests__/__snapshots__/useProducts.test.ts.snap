// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test useProducts hook should return products list 1`] = `
[
  {
    "category": "Электроника",
    "description": "Latest iphone, buy it now",
    "id": 1,
    "imgUrl": "/iphone.png",
    "name": "IPhone 14 Pro",
    "price": 999,
    "priceSymbol": "$",
  },
  {
    "category": "Одежда",
    "description": "Запускаем гуся, работяги",
    "id": 2,
    "name": "Костюм гуся",
    "price": 1000,
    "priceSymbol": "₽",
  },
  {
    "category": "Для дома",
    "description": "Говорят, что ее использовали в pixar",
    "id": 3,
    "imgUrl": "/lamp.png",
    "name": "Настольная лампа",
    "price": 699,
  },
  {
    "category": "Электроника",
    "description": "Незаменимая вещь для студента",
    "id": 4,
    "name": "Принтер",
    "price": 7000,
  },
]
`;
